global/js/main.js

```diff
@@ -190,28 +221,13 @@ function updateTheme() {
                 document.getElementById("css_cont").innerHTML += "<link id='" + theme + "_relative' rel='stylesheet' type='text/css' href='css/" + theme["css"][i] + "?v=" + version + "'>";
             }
         }
+        if (theme["customAccent"] != undefined) {
+            document.getElementById("custom_theme_container").innerHTML = `body {` + generateCustomThemeVars(theme["customAccent"].light, theme["customAccent"].dark, theme["customAccent"].lightOffset, theme["customAccent"].darkOffset) + `}`
+        }
     }
 
     if (theme.internal == "custom" || theme.internal == "custom-light") {
-        var accentDark_split = String(customTheme.accent_dark).split(",");
-        var accent_split = String(customTheme.accent).split(",");
-        var accentDark_hsl = colours.RGBToHSL(accentDark_split[0], accentDark_split[1], accentDark_split[2]);
-        var accent_hsl = colours.RGBToHSL(accent_split[0], accent_split[1], accent_split[2]);
-
-        document.getElementById("custom_theme_container").innerHTML = `html{
-            --accentdark: ${customTheme.accent_dark} !important;
-            --accent: ${customTheme.accent} !important;
-            
-            --accentdark_hue: ${accentDark_hsl[0]}deg;
-            --accent_hue: ${accent_hsl[0]}deg;
-            --accentdark_saturation: ${accentDark_hsl[1]}%;
-            --accent_saturation: ${accent_hsl[1]}%;
-            --accentdark_value: ${accentDark_hsl[2]}%;
-            --accent_value: ${accent_hsl[2]}%;
-
-            --accentdark_valueoffset: ${accentDark_hsl[2] / 45 + 0.2};
-            --accent_valueoffset: ${accentDark_hsl[2] / 50 + 0.2};
-        }`;
+        document.getElementById("custom_theme_container").innerHTML = `body {` + generateCustomThemeVars(customTheme.accent, customTheme.accent_dark) + `}`

```

Comment:
if you are already using ` just use the ${} feature on them, it looks a lot cleaner
```suggestion
        document.getElementById("custom_theme_container").innerHTML = `body {${generateCustomThemeVars(customTheme.accent, customTheme.accent_dark)}}`

```